src/Menu.php

```diff
@@ -84,6 +86,32 @@ public function addOptions(array $options): Menu
         return $this;
     }
 
+    /**
+     * Adds a new text input option.
+     *
+     * @param string $label
+     * @param string $promptText
+     * @param string $placeholder
+     * @return Menu
+     */
+    public function addTextInputOption(string $label, string $promptText, string $placeholder = 'Enter text...'): Menu
```

Comment:
Do you agree that, in the end, this is a question right?

What do you think about `addQuestion` instead?

Shouldn't the placeholder be null by default?

Reply:
Agreed.
Null not possible (respecting interface), so I put empty string.